editor.planx.uk/src/pages/FlowEditor/lib/store/preview.ts

```diff
@@ -885,3 +871,43 @@ export const removeNodesDependentOnPassport = (
   }, [] as string[]);
   return { removedNodeIds, breadcrumbsWithoutPassportData: newBreadcrumbs };
 };
+
+const collectedFlagValuesByCategory = (
```

Comment:
No logical change here - this has simply been extracted out of `autoAnswerableFlag` so we can re-use it in `collectedFlags` and `resultData` too